src/main/java/qna/domain/Answers.java

```diff
@@ -0,0 +1,40 @@
+package qna.domain;
+
+import org.hibernate.annotations.Where;
+import org.springframework.util.CollectionUtils;
+import qna.CannotDeleteException;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Embeddable
+public class Answers {
+
+    @OneToMany(mappedBy = "question", cascade = CascadeType.ALL)
+    @Where(clause = "deleted = false")
+    @OrderBy("id ASC")
+    private List<Answer> answers = new ArrayList<>();
+
+    public Answers() {
+    }
+
+    public void delete(User loginUser) throws CannotDeleteException {
+        for(Answer answer : answers) {
+            answer.delete(loginUser);
+        }
+    }
+
+    public boolean isEmpty() {
+        return CollectionUtils.isEmpty(answers);
+    }
+
+    public List<Answer> getAnswers() {
+        return answers;
```

Comment:
컬렉션을 그대로 반환하면 변조를 할 수 있어 일급 컬렉션의 의미가 없어지는 것 같아요.
컬렉션을 수정하지 못하는 타입을 반환하면 어떨까요?
```suggestion
        return Collections.unmodifiableList(answers);
```

Reply:
아 네 피드백 받은 부분이었는데 잊어먹었네요 .. 감사합니다!